instrumentation/liberty/liberty-dispatcher/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/liberty/dispatcher/LibertyDispatcherNetAttributesGetter.java

```diff
@@ -20,7 +20,7 @@ public String transport(LibertyRequest libertyRequest) {
   @Override
   @Nullable
   public Integer peerPort(LibertyRequest libertyRequest) {
-    return libertyRequest.getServerPort();
+    return libertyRequest.peerPort();
```

Comment:
should we also remove `getServerPort` method?

Reply:
I think we should probably start capturing `net.host.*` attributes, but may be worth waiting for the proposed `net.*` attribute changes to go through the spec first, I'll open an issue to track